transactions/lockedTokens/user/get_total_balance.cdc

```diff
@@ -37,7 +37,6 @@ pub fun main(address: Address): UFix64 {
         let nodeInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeStakerRef.id)
         sum = sum 
               + nodeInfo.tokensStaked
-              + nodeInfo.totalTokensStaked
```

Comment:
Why was this removed?

Reply:
That is a derived field so it doesn't actually reflect the total balance. I saw that is shouldn't be there while doing these and thought I would just get rid of it in the same PR